### DISCUSSION TITLE
“基础连接已经关闭：发送时发生错误”“由于远程方已经关闭传输流，身份验证失败”是啥意思啊？求大佬指点。

### DISCUSSION THREAD

**Discussion by huxinyan116** [shadowsocks/shadowsocks-windows#3270] [Q&A]
windows电脑端已经稳定使用一年了，账户过期重新续费之后电脑端登不上去了。安卓还可以用。日志显示“基础连接已经关闭：发送时发生错误”“由于远程方已经关闭传输流，身份验证失败”。请问是什么原因呢？

**Comment by chenshaoju** [comment]
如果你是购买的服务，请与你的服务商联系。
如果你是自己搭建的服务，请尝试套个插件：https://github.com/shadowsocks/shadowsocks-windows/issues/2228#issuecomment-465047836。
